backend/btrixcloud/operator.py

```diff
@@ -210,6 +211,7 @@ async def sync_crawls(self, data: MCSyncData):
             started=data.parent["metadata"]["creationTimestamp"],
             stopping=spec.get("stopping", False),
             expire_time=from_k8s_date(spec.get("expireTime")),
+            max_crawl_size=configmap["MAX_CRAWL_SIZE"],
```

Comment:
```suggestion
            max_crawl_size=configmap,get("MAX_CRAWL_SIZE", 0)
```

It's possible for this not to exist on old older configmaps that were created before this property is set.